pom.xml

```diff
@@ -109,6 +109,12 @@
             <version>2.5</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.ocpsoft.prettytime</groupId>
+            <artifactId>prettytime</artifactId>
+            <version>3.2.7.Final</version>
+        </dependency>
```

Comment:
What to add another library? This solution meets the requirements, the format is legible.

Reply:
This library is supporting a lot of locales so that it will be easier to localize this app